### ISSUE TITLE
How to specify the "name" for AccessibleNumberTweaker

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/area-model-common#150]
It looks like I'm specifying an `innerContent` for NumberPickers, but AccessibleNumberTweaker looks like it expects `labelContent` (since it specifies a `labelTagName`).

I'm not sure if this is the source of the empty paragraph in the current parallel DOM:
<img width="401" alt="screen shot 2018-05-24 at 10 38 48 am" src="https://user-images.githubusercontent.com/480878/40499287-bfb1cb72-5f3e-11e8-9fea-568f76b7ba0e.png">

Thoughts?

**Comment by zepumph** [comment]
The last @jessegreenberg and I talked about this, it seemed like `AccessibleNumberTweaker` "sounded good," but @jessegreenberg don't you think it needs inner content for an accessible name also? Assigning over to you because you implemented this originally, and may know better how changes would effect other usages like in GFLB.

**Comment by jessegreenberg** [comment]
Alright, I think we just finalized the representation for AccessibleNumberTweaker in https://github.com/phetsims/gravity-force-lab-basics/issues/62. We discovered that some AT really don't like inner content on things with `role="spinbutton"`. We had a workaround with aria-live, but that was also causing problems so now we are using aria-labelledby for the accessible name.

So to set the accessibleName for AccessibleNumberTweaker, one should use `labelContent`.

This is a another case where a consistent API for "accessible name" (hiding the implementation details) would be very beneficial.
